frontend/globals/viewer.ts

```diff
@@ -508,7 +513,14 @@ export class Viewer {
 
 		this.emit(Viewer.EVENT.START_LOADING);
 
-		UnityUtil.loadModel(this.account, this.model, this.branch, this.revision);
+		UnityUtil.loadModel({
```

Comment:
1. Object is more flexible.
2. Which value do you mean?

Reply:
1. Why does it need to be more flexible? You're depicting the exact values of the parameters for this function call anyway? I'm not a fan of wrapping object around parameters unless justified. Its makes things more vague and bypasses build checks.

2. I mean the loadModel returns a promise, this is basically related to the comment on the unity-utils.ts bit :)

Reply:
1. At least because of few reasons:
- Forcing naming convention
- Non-ordering of specific arguments
- Shorter function's head

These reasons are probably not so important for you to make the change like this. I'll change it. ;)
Just because I'm curious. How does it bypass build checks?

2. Ok!